build.sh

```diff
@@ -172,6 +172,9 @@ if buildAll || hasArg libcudf; then
         echo "Building for *ALL* supported GPU architectures..."
     fi
 
+    # get the current count before the compile starts
+    FILES_IN_CCACHE=$(ccache -s | grep "files in cache")
```

Comment:
There is no requirement for users of `build.sh` to have ccache installed. They might be using `sccache` or no caching at all. We should only execute this request if ccache is a valid command.